spec.html

```diff
@@ -5745,10 +5745,10 @@ <h1>
       <emu-alg>
         1. If _argument_ is *"-0"*, return *-0*<sub>𝔽</sub>.
         1. Let _n_ be ! ToNumber(_argument_).
-        1. If SameValue(! ToString(_n_), _argument_) is *false*, return *undefined*.
-        1. Return _n_.
+        1. If SameValue(! ToString(_n_), _argument_) is *true*, return _n_.
+        1. Return *undefined*.
       </emu-alg>
-      <p>A <em>canonical numeric string</em> is any String value for which the CanonicalNumericIndexString abstract operation does not return *undefined*.</p>
+      <p>A <dfn>canonical numeric string</dfn> is any String value for which the CanonicalNumericIndexString abstract operation does not return *undefined*.</p>
```

Comment:
```suggestion
      <p>A <dfn variants="canonical numeric strings">canonical numeric string</dfn> is any String value for which the CanonicalNumericIndexString abstract operation does not return *undefined*.</p>
```

as long as we're dfn'ing things.

Reply:
Done.